main-core/Mandarine.ns.ts

```diff
@@ -797,9 +797,8 @@ export namespace Mandarine {
             return Mandarine.Global.getNativeComponentsRegistry().get(MandarineCore.NativeComponents.WebMVCConfigurer).getSessionContainer();
         };
 
-        export const MandarineDefaultCorsOptions: Mandarine.MandarineMVC.CorsMiddlewareOption = {
-            origin: '*',
-            methods: ["GET", "HEAD", "PUT", "PATCH", "POST", "DELETE"],
+        export const MandarineDefaultCorsOptions: { optionsSuccessStatus: number} = {
```

Comment:
I want to keep the default configuration as [Express CORS](https://expressjs.com/en/resources/middleware/cors.html#configuration-options) . 
![image](https://user-images.githubusercontent.com/60560109/113027512-2c38c680-9158-11eb-9581-ff98290af734.png) . I think it's worth keeping that default configuration as is, adding the optionSuccessStatus. Also, let's not make anonymous types, so let's use `Mandarine.MandarineMVC.CorsMiddlewareOption` instead of ` { ... } ` and if needed, let's add more stuff to `Mandarine.MandarineMVC.CorsMiddlewareOption`